litevideo/csc/common.py

```diff
@@ -18,13 +18,21 @@ def saturate(i, o, minimum, maximum):
 def coef(value, cw=None):
     return int(value * 2**cw) if cw is not None else value
 
-
 def rgb_layout(dw):
     return [("r", dw), ("g", dw), ("b", dw)]
 
+def rgb16f_layout(dw):
+    return [("r_f", dw), ("g_f", dw), ("b_f", dw)]
 
 def ycbcr444_layout(dw):
     return [("y", dw), ("cb", dw), ("cr", dw)]
 
 def ycbcr422_layout(dw):
     return [("y", dw), ("cb_cr", dw)]
+
+def pix_layout(dw):
+    return [("pix", dw)]
+
+def pixf_layout(dw):
+    return [("pixf", dw)]
```

Comment:
pixf is not consistent with r_f, g_f, b_f (no underscore). Also I'm wondering if we really need the _f suffix in the layout, maybe better just use r, g, b, pix since we'll already be aware that we are working with floating elements.